wolfssl/tests/ping_pong.rs

```diff
@@ -192,18 +193,27 @@ async fn server<S: SockIO>(sock: S, protocol: Protocol) {
     println!("[Server] Finished");
 }
 
-#[tokio::test]
-async fn dtls() {
-    use Protocol::*;
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_3 => panics; "client_1.2_server_1.3")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_2; "client_1.2_server_1.2")]
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServer; "client_1.2_server")]
+#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServerV1_3; "client_1.3_server_1.3")]
+#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServerV1_2 => panics; "client_1.3_server_1.2")]
+#[test_case(Protocol::DtlsClientV1_3, Protocol::DtlsServer; "client_1.3_server")]
+#[test_case(Protocol::DtlsClient, Protocol::DtlsServerV1_3; "client_server_1.3")]
+// TODO: WolfSSL downgrade bug
+// #[test_case(Protocol::DtlsClient, Protocol::DtlsServerV1_2; "client_server_1.2")]
+#[test_case(Protocol::DtlsClient, Protocol::DtlsServer; "client_server")]
 
```

Comment:
nit: You don't need a blank line between the cases and the tokio tag.

Reply:
Yep, I know. I added it for readabiliy.
But cargo fmt does not like that and removed it anyway. So it s not there in the final commit.